web/router.ex

```diff
@@ -18,6 +18,7 @@ defmodule Healthlocker.Router do
 
     get "/", PageController, :index
     resources "/posts", PostController, only: [:show, :new, :create]
+    resources "/users", UserController, only: [:show, :new, :create, :index, :edit, :delete, :update]
```

Comment:
If you're including all the actions then you probably don't need the `only: ` list, just `resources` will generate then all for you.